plugins/source/test/client/spec.go

```diff
@@ -1,13 +1,23 @@
 package client
 
 type Spec struct {
-	NumClients int `json:"num_clients"`
+	NumClients int  `json:"num_clients"`  // Number of clients to create. Default: 1
+	NumRows    *int `json:"num_rows"`     // Number of rows to generate in test_some_table. Defaul: 1
```

Comment:
make this simple `int` values & update `<=0` to `1` in `SetDefaults`

Reply:
No we want 0 to be available for no rows.